.github/workflows/linux.yml

```diff
@@ -0,0 +1,71 @@
+name: Linux tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+
+env:
+  # Increase this value to reset cache if ci/environment.yml has not changed
+  CACHE_NUMBER: 1
+
+
+jobs:
+  Build:
+    strategy:
+      matrix:
+        include:
```

Comment:
If we test only one OS, then there is no need for a matrix. Instead, we can use like this: https://github.com/marketplace/actions/setup-miniconda#caching-packages

Reply:
I'll do those clean-ups later on. Trying to get it run once.